pkg/gzip/gzip.go

```diff
@@ -165,3 +191,11 @@ func ExtractTarGz(owner string, fp io.Reader, dest string) (err error) {
 	}
 	return nil
 }
+
+func firstExistingParent(path string) string {
+	for _, err := os.Stat(path); os.IsNotExist(err); {
+		path = filepath.Dir(path)
+		_, err = os.Stat(path)
```

Comment:
above in `os.IsNotExist(err)`.

I could move this update to be after the second semicolon to make that more clear that it's the same `err`. A bit undecided.